ArcGISRuntimeSDKQt_QMLSamples/Layers/ApplyUniqueValuesWithAlternateSymbols/README.md

```diff
@@ -16,23 +16,22 @@ Zoom in and out of the map to see alternate symbols at each scale. To go back to
 
 ## How it works
 
-This sample uses a unique value renderer that is created with a unique value that has a list of alternate symbols and their corresponding reference properties to specify at which scales each symbol should appear. Once a unique value renderer with alternate sybmols is created, it is assigned as the renderer of the feature layer.
+1. Create a `featureLayer` using the service url and add it to the map's list of operational layers.
+2. Create two alternate symbols (a blue square and a yellow diamond) to be used as alternate symbols. To create an alternate symbol:
 
-1. Create a feature layer using the service url and add it to the list of operational layers.
-2. Create two alternate symbols (a blue square and a yellow diamond) to be used for the unique value. To create an alternate symbol:
     a. Create a symbol using `SimpleMarkerSymbol`.
-    b. Convert the Simple Marker Symbol to a Multilayer Symbol.   
-    c. Set the valid scale range through reference properties on the multilayer point symbols blue square and yellow diamond by calling `SymbolReferenceProperties`.
     
-3. Create a third multilayer symbol to be used when creating the unique value:
-    a. Create a red triangle and set the reference properties.
+    b. Convert the simple marker symbol to a `MultilayerSymbol` using `SimpleMarkerSymbol::toMultilayerSymbol`.
     
-4. Create a unique value using the red triangle from step three and the list of alternate symbols from step two.
-5. Create a unique value renderer and add the unique value from step four to it.
+    c. Set the valid scale range through reference properties on the multilayer point symbols blue square and yellow diamond by calling `MultilayerSymbol::setReferenceProperties(new SymbolReferenceProperties(minScale, maxScale));`.
```

Comment:
As above, should `MultilayerSymbol::setReferenceProperties(new SymbolReferenceProperties(minScale, maxScale));` be changed to something like `MultilayerSymbol.referenceProperties` to make it look more like Qml?